breathing_cat/build.py

```diff
@@ -769,6 +769,10 @@ def build_documentation(
     except FileNotFoundError:
         pass  # simply don't add general documentation if no doc files exist
 
+    # Link project_source_dir as PKG in build directory.  This allows, for example, to
+    # include source files from the package using a path like `/PKG/scripts/foo.py`.
+    os.symlink(project_source_dir, doc_build_dir / "PKG", target_is_directory=True)
```

Comment:
Will that work on none Linux (windows) machines?
Just to be curious...

Reply:
To be honest, I don't know.  At least creating the symlink should work under Windows as well.  I don't know if it then can be used in the same way or if it would behave differently.
It is to some degree covered by the unit tests.  I can extend them here on GitHub to also run on MacOS and Windows, this would give us some idea (will do this in a separate PR).

Reply:
no worries, we will figure this out later